component-studio/component-runtime-di/src/main/java/org/talend/sdk/component/runtime/di/record/DiRowStructVisitor.java

```diff
@@ -435,6 +435,9 @@ private void onArray(final Entry entry, final Collection value) {
     }
 
     private void onObject(final String name, final Object value) {
+        // in studio di, the object value is suitable to json string always? this code for construct the input tck
+        // record to tck processor, maybe is ok
+        // will adjust it if not meet real requirement
         recordBuilder.withString(name, jsonb.toJson(value));
```

Comment:
may not fit against what is done in the PR...

Reply:
need to fix here for jdbc blob/clob type which map to id_Object, for example, suppose this all tck connector:
tmysqlinput(blob column)==>toracleoutput(blob column)
we can't call json.tojson here.